src/lib/helpers.ts

```diff
@@ -23,6 +23,27 @@ export function getParameterByName(name: string, url?: string) {
   return decodeURIComponent(results[2].replace(/\+/g, ' '))
 }
 
+// #107 Only remove Gotrue parameters from url hash.
+export function clearGotrueHashParameters() {
+  window.location.hash = removeParamsFromUrl(window.location.hash, [
```

Comment:
Also, just a question here, would it be possible to rather get the parameters from the Session object? This will make sure that should there be any new property added or property removed, this would always work.

i.e. `Object.keys(session)` and then pass them to the function

Reply:
Yes good idea, I didnt' think about it.